streams/src/test/java/org/apache/kafka/streams/processor/internals/GlobalProcessorContextImplTest.java

```diff
@@ -82,20 +79,12 @@ public void setup() {
             null,
             null);
 
-        final ProcessorNode<?, ?> processorNode = mock(ProcessorNode.class);
-        globalContext.setCurrentNode(processorNode);
+        final ProcessorNode<Object, Object, Object, Object> processorNode = new ProcessorNode<>("testNode");
 
         child = mock(ProcessorNode.class);
+        processorNode.addChild(child);
 
-        expect(processorNode.children())
```

Comment:
Ah, it's because I swapped out the mock of the ProcessorNode for a real processor node (on line 82). Note that these were not checks, they were just dummy returns, since we never verified them, and it was a nice mock.

Now, all the tests behave the same, and we don't need as much boilerplace test setup.